src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salary;
+        int hours;
+        LocalDate fromDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate toDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder stringBuilder =
+                new StringBuilder("Report for period ")
+                        .append(dateFrom).append(" - ").append(dateTo);
+        for (String string : names) {
+            int sumSalary = 0;
+            String name;
+            LocalDate date;
+            for (String datum : data) {
+                name = datum.split(" ")[1];
```

Comment:
let's create an array and `split()` into it once to not split every time we need data